### DISCUSSION TITLE
Directus v9: ?access_token=<token> don't work.

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#9571] [Archive]
I try to use localhost:8055/items/diary/?access_token=<token> where token is configured into the token field, but I've alway 403 error.
Is't still valid this method for the Directus v9?

**Comment by rijkvanzanten** [comment]
Does the collection `diary` exist? Does the user you're authenticating as have permissions to the `diary` collection? Can you try omitting the trailing slash in that endpoint (eg use `../diary?..` instead of `../diary/?..`

**Reply by biapar** [reply]
Yes. The collection exist. I tried with and without /. The user has the permission on the collection.

**Comment by biapar** [comment]
Solved. The name of collection is case sensitve. Diary is different from diary.
This works.
localhost:8055/items/Diary?access_token=ZfVYXR42HS9KdkclkxlkcxlkcxeeeXwf7iB2QGtn80Az3qZ
